ORBIT/phases/install/cable_install/array.py

```diff
@@ -149,13 +170,53 @@ def install_array_cables(
     ----------
     vessel : Vessel
         Cable installation vessel.
-    cables : list
+    cable_data : list
         List of tuples containing `Cable` instances and sections.
     burial_vessel : Vessel
         Optional configuration for burial vessel. If configured, the
         installation vessel only lays the cable on the seafloor and this
         vessel will bury them at the end of the simulation.
+    trench_vessel: Vessel
+        Optional configuration for trenching vessel.  If configured, the
+        trenching vessel travels along the cable route prior to arrival of
+        the cable lay vessel and digs a trench.
     """
+    # print(cable_data)
+    # for c,s in cable_data:
+    #     print(c,s)
+    # Conduct trenching along cable routes before laying cable
+    if trench_vessel is None:
+        pass
+    else:
+        # Create list of cable section lengths to dig trenches for
+        total_trench_distance = []
+
+        for cable, sections in cable_data:
+            vessel.cable_storage.reset()
```

Comment:
This isn't needed if the vessel isn't carrying any cable.

Reply:
Same here.

Reply:
ok, i wasn't sure about that one.